### DISCUSSION TITLE
Sending welcome email from Wiki.js

### DISCUSSION THREAD

**Discussion by pawllk** [requarks/wiki#7180] [Help / Questions]
Hello,
Currently, I use wiki.js 2.5.303 at work. For a long time I have been struggling with sending welcome emails when a user is created. During creation, I see that the "Send welcome message" option is grayed out and not available. From the information I have read, this thread already appeared in 2017. Does anyone know how to enable this functionality or is it available?
![Wiki](https://github.com/requarks/wiki/assets/72317779/2aabb8f6-7ee4-4488-9d49-4a42ae82c712)

**Comment by WaliSW** [comment]
Hi,
I also need this functionality in my environment, it's urgent for me.
